src/models/Room.js

```diff
@@ -0,0 +1,104 @@
+const mongoose = require('mongoose')
+const Joi = require('joi')
+const { PREFLOP } = require('../constants')
+
+
+const roomSchema = new mongoose.Schema({
+    players: {
+        type: Array,
+        required: true
+    },
+    maxPlayers: {
+        type: Number,
+        required: true
+    },
+    name: {
+        type: String,
+        required: false
+    },
+    hand: {
+        type: Array,
+        required: false
+    },
+    maxBuyIn: {
+        type: Number,
+        required: true
+    },
+    bigBlind: {
+        type: Number,
+        required: true
+    },
+    smallBlind: {
+        type: Number,
+        required: true
+    },
+    bets: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    lastToRaiseId: {
+        type: String,
+        required: false
+    },
+    pot: {
+        type: Number,
+        required: true,
+        default: 0
+    },
+    playersWaiting: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    phase: {
+        type: String,
+        required: true,
+        default: PREFLOP
+    },
+    deck: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    communityCards: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    allInHands: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    sidePots: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    winners: {
+        type: Array,
+        required: true,
+        default: []
+    },
+    endedByFold: {
+        type: Boolean,
+        required: true,
+        default: false
+    },
+    numBots: {
+        type: Number,
+        required: true,
+        default: 0
+    },
+    moveHistory: {
+        type: Array,
+        default: []
+    }
```

Comment:
Do you actually need to write all this? 
```suggestion
    ...gameSchema
```

Assuming gameSchema can be extracted from models/Game somehow. Or maybe you can export the gameSchema as well from the Game file.

Reply:
got it, removed all of this and imported gameSchema and used it